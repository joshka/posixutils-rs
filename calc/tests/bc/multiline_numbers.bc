1\
2

123\
456

0.\
345

0\
.456

quit
