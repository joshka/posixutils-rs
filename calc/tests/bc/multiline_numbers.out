12
123456
0.345
0.456
